src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -212,9 +232,27 @@ angular.module("ovh-angular-mondial-relay")
                     this.loading.search = true;
                     this.ngModel = null;
                     this.foundRelays = [];
+                    var parsedFilter = filter;
+
+                    if (!parsedFilter) {
+                        parsedFilter = this.filter;
+                        var searchQuery = _.get(parsedFilter, "searchQuery", "");
+                        if (searchQuery) {
+                            var zipcode = _.compact(searchQuery.match(/\d{5}/g));
+                            if (zipcode.length) {
+                                parsedFilter.zipcode = _.first(zipcode).trim();
+                            } else {
+                                var city = _.compact(searchQuery.match(/[A-z\s-]*/g));
```

Comment:
it doesn't work city name with special chars (àé...) or quote (')

eg.

"L'Abergement-Sainte-Colombe".match(/[A-z\s-]*/g)
(4) ["L", "", "Abergement-Sainte-Colombe", ""]

"Lès".match(/[A-z\s-]*/g)
(4) ["L", "", "s", ""]